libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -148,7 +148,8 @@
     border: 1px solid var(--feds-borderColor--light);
     background: var(--feds-background-nav--light);
     white-space: normal;
-    box-sizing: content-box;
+    box-sizing: border-box;
+    padding: 24px 32px;
```

Comment:
This also squeezes the promo image. Image should not have any paddings, just the text content. This also brings the lateral padding of the text to `64px`, when it should be just `32px`.

In the screenshot below, we want the image to keep behaving like the original implementation. Just the text box should be adapted.

<img width="513" alt="Screenshot 2023-11-21 at 12 03 18" src="https://github.com/adobecom/milo/assets/11267498/b891fd8d-9761-4dea-ae8e-ec7bb4b35a2d">

Reply:
Solved